doc/whats-new.rst

```diff
@@ -34,6 +34,10 @@ v0.10.4 (unreleased)
 Enhancements
 ~~~~~~~~~~~~
 
+- Support reading and writing lists of strings as attributes to netCDF
```

Comment:
Minor nit: this should be "Support writing lists" only. We can already read lists of strings.